src/certificateForm/FormCertificate.js

```diff
@@ -12,12 +13,14 @@ const StyledForm = styled.form`
 `
 
 export default function FormCertificate({ onFormSubmit, history }) {
-  const [certificate, setCertificate] = useState({})
+  const [certificate, setCertificate] = useState({
```

Comment:
does this state only include the date? if so, maybe rename the state

Reply:
No, but I initialize it with the date as a default value, later the id too.